### DISCUSSION TITLE
Hi! Where is the documentation repository for this tool?

### DISCUSSION THREAD

**Discussion by peterroe** [x-cmd/x-cmd#59] [General]
First, Thanks this project !!!

[This document](https://x-cmd.com) has many awesome effects, but I did not find the code repository corresponding to the document. Is it closed source?

**Comment by edwinjhlee** [comment]
> First, Thanks this project !!!
> 
> [This document](https://x-cmd.com) has many awesome effects, but I did not find the code repository corresponding to the document. Is it closed source?

Thanks.

x-cmd is open source project. 
https://github.com/x-cmd/x-cmd/tree/main

But the website is not an open source project.

**Comment by peterroe** [comment]
Oh, What a pity. Do you have any plans to open source the documentation?

**Reply by edwinjhlee** [reply]
We will consider. Thank you.
